crates/indexer/src/processors/token_processor.rs

```diff
@@ -180,7 +180,7 @@ fn insert_tokens(
                 .on_conflict((token_data_id_hash, property_version, transaction_version))
                 .do_update()
                 .set((
-                    collection_data_id_hash.eq(excluded(collection_data_id_hash)),
+                    token_properties.eq(excluded(token_properties)),
```

Comment:
what is this `eq(excluded(token_properties))` used for?

Reply:
this is just to override existing token_properties. Actually since we backfilled I can probably remove it...but i generally just remove it next time.